### ISSUE TITLE
Impossible de rendre des champs du dénombrement optionnel

### ISSUE THREAD

**Issue by gildeluermoz** [PnX-SI/gn_mobile_occtax#204]
### Version de l'application

Version d'Occtax-mobile affectée par le bug : 2.4.0
Version de GeoNature utilisée : 2.11.2

### Terminal et Version Android

Marque et modèle du terminal : Samsung Galaxy S7
Version d'Android : 8

### Description du bug et comportement attendu
Lorsque l'on souhaite mettre certains champs du dénombrement derrière un bouton "dénombrement avancé" avec la configuration ci-dessous, rien ne change: tous les champs sont au même niveau et il n'y a pas de bouton "dénombrement avancé" qui apparaît.
Ce comportement qui fonctionne pour la partie information. Peut-être que ce comportement n'est pas prévu dans la partie dénombrement. Dans ce cas il faut préciser dans la doc que seul l'affichage (true ou false) du champ est modifiable dans la conf.
```
"counting": [
            {
                "key" : "STADE_VIE",
                "visible": true,
                "default": true
            },
            {
                "key" : "SEXE",
                "visible": true,
                "default": true
            },
            {
                "key" : "OBJ_DENBR",
                "visible": true,
                "default": false
            },
            {
                "key" : "TYP_DENBR",
                "visible": true,
                "default": false
            },
            {
                "key" : "MIN",
                "visible": true,
                "default": true
            },
            {
                "key" : "MAX",
                "visible": true,
                "default": true
            }
        ]
```

### Comment reproduire

mettre la config proposée ci-dessus dans le settings.json dans le but de mettre  objet du dénombrement et type de dénombrement derrière un bouton "avancé".

**Comment by camillemonchicourt** [comment]
Oui, de mémoire, ce comportement n'est pas prévu dans la partie DENOMBREMENT.

**Comment by gildeluermoz** [comment]
Tu vois pour adapter la doc ? Peut-être voir ce qu'en dit @sgrimault .

**Comment by sgrimault** [comment]
Bonjour,
Cette possibilité n'est disponible que sur la partie information.
